react/README.md

```diff
@@ -34,12 +34,28 @@ For example, the following works:
 <MaskedTextInput
   mask={['(', /[1-9]/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]}
   className="form-control"
-  placeholder="Enter zip code"
+  placeholder="Enter a phone number"
   guide={false}
   id="my-input-id"
 />
 ```
 
+You can additionally use your own custom input component by passing in the prop `inputElement`.
```

Comment:
How about we make the user responsible for passing in a ref to an HTML input element?